lib/modules/platform/gitea/index.md

```diff
@@ -18,7 +18,10 @@ You must set `platform=gitea` in your Renovate config file.
 The PAT should have these permissions:
 
 - `repo`
-- `read:user` and `read:email`
+- `read:user`
```

Comment:
Can we try a table layout instead? I'm thinking of something like this:

| Permission   | Valid for Gitea versions       |
| ------------ | ------------------------------ |
| `repo`       | all                            |
| `read:user`  | all                            |
| `read:email` | `< 1.19.3`                     |
| `issue`      | `> 1.20.0`                     |
| `read:misc`  | Only for `1.20.0` and `1.20.1` |

Reply:
Love the idea of a table layout. Looks cleaner like this. Done in https://github.com/renovatebot/renovate/pull/23615/commits/36c3af5e4da57cbc5be1c835b1e94e6f85f97cca . I just slightly changed the order to have the PAT scopes for newer versions above the ones for older versions.